learn-accessibility-by-building-a-website-for-people-with-disabilities/index450.html

```diff
@@ -0,0 +1,138 @@
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width">
+    <title>CodeHouse — Take Our Survey</title>
+    <link rel="stylesheet" href="css/style220.css">
+  </head>
+
+  <body>
+    <header class="top-bar">
+      <a class="skip-nav-link" href="#main-content">Skip to main content</a>
+      <nav class="top-nav">
+        <ul>
+          <li>
+            <a class="nav-link" href="#">Home</a>
+          </li>
+          <li>
+            <a class="nav-link" href="#">Events</a>
+          </li>
+          <li>
+            <a class="nav-link" href="#">About</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <div class="outer">
+      <nav class="secondary-nav">
+        <h1 class="secondary-nav-heading">Recent Events</h1>
+        <ul>
+          <li>
+            <a href="#">
+              <time datetime="2019-04-04">Thu, 04 Apr 2019</time>
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <time datetime="2019-03-28">Thu, 28 Mar 2019</time>
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <time datetime="2019-03-21">Thu, 21 Mar 2019</time>
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <time datetime="2019-03-14">Thu, 14 Mar 2019</time>
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <time datetime="2019-03-07">Thu, 07 Mar 2019</time>
+            </a>
+          </li>
+        </ul>
+      </nav>
+
+      <main id="main-content" class="main-content">
+        <h1 class="top-heading">Take our survey!</h1>
+        <img alt="One of our students coding a Tic Tac Toe game" src="assets/code-coder-coding-34649.jpg" class="image">
+        <form class="form">
+
+          <div class="form-group">
+            <fieldset>
+              <legend class="field-label">
+                Where did you find out about CodeHouse?
+              </legend>
+              <input id="google" name="where" type="radio"> <label for="google">Google search</label>
+              <input id="friend" name="where" type="radio"> <label for="friend">Word of mouth</label>
+              <input id="other1" name="where" type="radio"> <label id="other-label" for="other1">Other (please specify)</label>
+              <input aria-labelledby="other-label" id="other2" type="text" class="form-control other">
+            </fieldset>
+          </div>
+
+          <div class="form-group">
+            <label for="how-many" class="field-label">
+              How many of our events have you attended in the past?
+            </label>
+            <input type="number" id="how-many" class="form-control">
+          </div>
+
+          <fieldset>
+            <legend>
+              Start, stop, keep
+            </legend>
+
+            <div class="form-group">
+              <!--
+                Give the elements the correct `id` and `for` elements, using
```

Comment:
Give the elements the correct `id` and `for` attributes, using the IDs "start", "stop", and "keep" respectively.